controllers/front/ProductController.php

```diff
@@ -1089,7 +1089,7 @@ protected function getProductMinimalQuantity($product)
             $minimal_quantity = $this->product->minimal_quantity;
         }
 
-        return $minimal_quantity;
+        return max($minimal_quantity, 1);
```

Comment:
Warning: this function is used in multiple places. Modifying its behavior can create side-effects (for example with the computations related to check whether user can order X amount on product A provided it has stock B).

So we better have extensive checks 🤔